src/openrct2/actions/FootpathRemoveAction.hpp

```diff
@@ -95,6 +95,17 @@ DEFINE_GAME_ACTION(FootpathRemoveAction, GAME_COMMAND_REMOVE_PATH, GameActionRes
             map_invalidate_tile_full(_loc);
             tile_element_remove(footpathElement);
             footpath_update_queue_chains();
+
+            // Remove the spawn point (if there is one in the current tile)
+            gPeepSpawns.erase(
+                std::remove_if(
+                    gPeepSpawns.begin(), gPeepSpawns.end(),
+                    [this](const CoordsXYZ& spawn) {
+                        {
+                            return spawn.ToTileStart() == _loc;
```

Comment:
If you are doing a compare like this use the same function on both. Can you change loc so that it uses to tile start.